test.ps1

```diff
@@ -0,0 +1,37 @@
+"Running tests"
+$ErrorActionPreference = "Stop"
+$version = $env:APPVEYOR_BUILD_VERSION -replace('\.[^.\\/]+$')
+
+"TEST: Version $version in docker.nuspec file should match"
+[xml]$spec = Get-Content docker.nuspec
+if ($spec.package.metadata.version.CompareTo($version)) {
+  Write-Error "FAIL: rong version in nuspec file!"
+}
+
+"TEST: Package should contain only install script"
+Add-Type -assembly "system.io.compression.filesystem"
+$zip = [IO.Compression.ZipFile]::OpenRead("$pwd\docker.$version.nupkg")
+if ($zip.Entries.Count -ne 5) {
+  Write-Error "FAIL: Wrong count in nupkg!"
+}
+$zip.Dispose()
+
+"TEST: Installation of package should work"
+. choco install -y docker -source .
```

Comment:
can you add that option as well. or maybe we should specify arch explicitly. I'm not sure if appveyor has 64 bit.